src/js/EditsQ029.jsx

```diff
@@ -0,0 +1,52 @@
+var React = require('react');
+var EditsDetail = require('./EditsDetail.jsx');
+
+var EditsQ029 = React.createClass({
+  propTypes: {
+    group: React.PropTypes.array
+  },
+
+  makeSelect: function() {
```

Comment:
Does this make more sense as a checkbox? A select for a yes/no seems weird to me